src/js/view/title.ts

```diff
@@ -74,10 +75,10 @@ class Title {
             let title = '';
             let description = '';
 
-            if (item.title && model.get('displaytitle')) {
+            if (typeof item.title === 'string' && model.get('displaytitle')) {
```

Comment:
Do we definitely need these `typeof` checks here? I'm not sure the implications of adding these and whether it'll result in `title` or `description` being blank in certain cases. Wondering if we can add the `typeof` checks into `HTMLSafeString`. Or maybe it's by design that we want to disallow any non-string values here?

Reply:
Yep. PlayListItem isn't strongly typed (it's imported from JS)